liquibase-integration-tests/src/test/groovy/liquibase/command/core/GenerateChangeLogDb2CommandTest.groovy

```diff
@@ -3,6 +3,7 @@ package liquibase.command.core
 import liquibase.Scope
 import liquibase.command.CommandResultsBuilder
 import liquibase.command.CommandScope
+import liquibase.command.core.helpers.DbUrlConnectionCommandStep
```

Comment:
Are these tests running anywhere? The `@ignore` annotation on the test class makes me think the answer is no. If that is the case then a) why isn't it running and b) can we make them run?

Reply:
Hello -> I found https://datical.atlassian.net/browse/DAT-13113 that was created to enable it.